quay-enterprise/configure-machines.md

```diff
@@ -1,15 +1,74 @@
-# Configure Machines for Enterprise Registry
+# Configure Machines for Quay Enterprise
 
-The Enterprise Registry allows you to create teams and user accounts that match your existing business unit organization. A special type of user, a robot account, is designed to be used programatically by deployment systems and other pieces of software. Robot accounts are commonly configured with read-only access to an organizations repositories.
+Quay Enterprise allows you to create teams and user accounts that match your existing business unit organization. A special type of user, a robot account, is designed to be used programatically by deployment systems and other pieces of software. Robot accounts are commonly configured with read-only access to an organizations repositories.
```

Comment:
Soft suggest:
Quay Enterprise allows you to create user accounts and *teams*, or groups, of those users that mirror your existing org chart.

...
s/organizations/organization's/
OR
"Robot accounts are usually configured with read-only access to a repository."